docs/install/README.md

```diff
@@ -529,9 +529,9 @@ RSSHub 支持使用访问密钥 / 码，白名单和黑名单三种方式进行
 
 ### 图片处理
 
-::: warning 废弃警告
+::: info 新配置方式
```

Comment:
Not tested?

![](https://user-images.githubusercontent.com/15956627/174869617-840b57a2-dbf1-496f-9a1b-5e74c0883171.png)